### ISSUE TITLE
Excute query call by user action

### ISSUE THREAD

**Issue by lukass77** [allegro/turnilo#1057]
Today working on Turnilo , end user set the query criteria on filter / split / musures , after end user done with query critiria ..
the query submited **automaticly** to druid broker .

is there is some way that user set his query criteria and only after pressing a button on UI / explicitly submit the query to broker , point is to provide the end user the controll to deciede when to submit the query .. instead executing it automaticly ..??

think of a flow where user want to add 5 musures , and 2 filter .
today for each filter added a query is submitted to druid broker .. and for each filter query submitted to druid broker ..
so we end with 7 calls to the broker .. but what user want is a single call to the broker which is the last combination of 5 measures and 2 filters 

can query execution can be controll by user action and not execute automaticly ?
if this is missing suggest to add this option , to configure turnilo .. with
env variable query_execution_mode , which can get two value - automaticly (default) as today , or user_action - user need to click a button on UI to execute the query after done setting query criteria .

**Comment by adrianmroz-allegro** [comment]
Duplicate of #1052
